lib/tasks/staging_seeds.rake

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+namespace :comfy do
+    desc "Import CMS Seed data from staging"
+
+    task :seed_import => :environment do |_t|
+      require 'net/ssh'
+      require 'net/scp'
+      
+      def check_timestamp(file, session, local_file, remote_file)     
+        newer = Time.at(file.attributes.mtime) >= File.stat(local_file).mtime 
+        if newer 
+          puts "#{file.name} is newer than the local copy, downloading..."
+          session.scp.download!(remote_file, local_file)
+        end
+      end
+      
+      # Locally stored seeds - assumes you already have the local folder - 
+      # it won't create it 
+      local = ComfortableMexicanSofa.config.seeds_path + '/protected-planet'
+      remote = 'ProtectedPlanet/current/db/cms_seeds/protected-planet'
+      
+      puts "Importing CMS Seed data from Staging Folder to #{local} ..."
+
+      # SSH into staging server with Net::SSH
+      Net::SSH.start(ENV['PP_STAGING'], ENV['PP_USER']) do |session|
+        session.sftp.dir.glob(remote, '**/*').each do |file|
```

Comment:
I wonder if we should consider pulling entire folders based on timestamps instead of files?
I mainly see 2 reasons for that:
 1. Reduce the number of scp commands
 2. Files can be deleted from staging so I don't think the script is taking this into account. This means that the system that is pulling data from staging might potentially store old and unnecessary files forever.

Reply:
True, I could just download the folders recursively instead of singly, but not sure I'm able to pass a block to the `download!` method as it needs to check the timestamps and also make sure the encoding is correct. And yes, the script isn't taking deleted files into account, I should really add that functionality in.

Reply:
I think that `download!` method might simply accept folder names as it does for files? In the same way you should be able to check timestamps for the folders. I believe that if something within a folder changes (e.g. another folder or a file), the timestamp of that folder changes too

Reply:
It does indeed, OK I'll just move all of that logic into the function and see how it goes.

Reply:
Task seems to work as before, but I've noticed something - we have two copies of every file in the `files` folder, one with a preceding underscore and one without. Is this something we want? I thought this was an error.

<img width="803" alt="Screenshot 2020-07-28 at 18 17 33" src="https://user-images.githubusercontent.com/53650048/88698939-9d723680-d0fe-11ea-8de5-81e0f28081a5.png">